packages/forms/test/form_builder_spec.ts

```diff
@@ -55,6 +55,59 @@ describe('Form Builder', () => {
     expect(g.controls['login'].asyncValidator).toBe(asyncValidator);
   });
 
+  it('should create controls from FormStates', () => {
+    const c = b.control({value: 'one', disabled: false});
+    expect(c.value).toEqual('one');
+    c.reset();
+    expect(c.value).toEqual(null);
+  });
+
+  it('should work on a directly constructed FormBuilder object', () => {
+    const g = new FormBuilder().group({'login': 'some value'});
```

Comment:
You can probably use `b` const (that refers to the `FormBuilder` instance) here:
```suggestion
    const g = b.group({'login': 'some value'});
```

Reply:
Done.